system/soundd.py

```diff
@@ -40,21 +40,28 @@
     loaded_sounds[sound] = f.read(dtype='float32')
 
 
-def soundd():
+def main():
   sm = messaging.SubMaster(['controlsState', 'microphone'])
```

Comment:
note that soundd isn't a `system` process since it subscribes to stuff in selfdrive